src/Types/Objects/ModelType.php

```diff
@@ -40,6 +40,7 @@ public function __construct(string $name, TypeLoader $typeLoader)
                     $args = [
                         'first' => Type::int(),
                         'page' => Type::int(),
+                        // TODO: replace typeloader with Type::get
                         'where' => $typeLoader->load('_' . $name . '__' . $key . 'EdgeWhereConditions'),
                         'orderBy' => new ListOfType(
```

Comment:
gibts hier nicht `Type::listOf`?
bzw gibts einen Grund für diese TODO?

Reply:
`Type::listOf()` gibt es, und das Ziel ist, nach und nach alle `new ListOfType()` damit zu ersetzen - aber es gibt noch viele Stellen wo das `new` immer noch im Einsatz ist, und ich bin nicht alles durchgegangen. Aber das kommt schon noch.

Das TODO ist deshalb da, damit ich über den Urlaub nicht vergesse, dass ich mir das noch ansehen muss. Aktuell kann das an dieser Stelle nicht auf `Type::getByName` geändert werden, weil dann aus noch ungeklärten Gründen im GraphQL ein Error fällt. Ich wollte aber nicht, dass der PR noch größer wird, und vorm Urlaub nicht fertig wird - darum würde ich das in einem getrennten PR machen.